### PR TITLE
sxiv: add page

### PR DESCRIPTION
**Pull request by MarkusG**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/sxiv.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`echo {{path/to/file}} | sxiv -i`
+
+- View images as a slideshow:
+
+`sxiv -S {{delay}} {{path/to/image}} ...`
+
+- View images as thumbnails:
+
+`sxiv -t {{path/to/image}} ...`
```
Comment (navarroaxel) [inline pages/linux/sxiv.md:24]:
```suggestion
`sxiv -t {{path/to/file}}`
```

File: pages/linux/sxiv.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- View an image or set of images, reading filenames from standard input:
+
+`echo {{path/to/file}} | sxiv -i`
+
+- View images as a slideshow:
+
+`sxiv -S {{delay}} {{path/to/image}} ...`
+
+- View images as thumbnails:
```
Comment (navarroaxel) [inline pages/linux/sxiv.md:22]:
```suggestion
- Open a space-separated list of images in thumbnails mode:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @MarkusG, I left some comments.

File: pages/linux/sxiv.md (hunk 2 above)
Comment (MarkusG) [reply pages/linux/sxiv.md]: Resolved in 9846d6a0e1471e8baad669860ac4c8c4092dbd5c

File: pages/linux/sxiv.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- View an image:
+
+`sxiv {{path/to/file}}`
+
+- View an image in fullscreen mode:
+
+`sxiv -f {{path/to/file}}`
+
+- View an image or set of images, reading filenames from standard input:
```
[the remark this answers is not in the document]
Comment (MarkusG) [reply pages/linux/sxiv.md]: Resolved in 9846d6a0e1471e8baad669860ac4c8c4092dbd5c

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks,  @MarkusG!

**Pull request merged**